esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1290,11 +1291,15 @@ bool WsWuInfo::getClusterInfo(IEspECLWorkunit &info, unsigned long flags)
     {
         int clusterTypeFlag = 0;
 
+#ifndef _CONTAINERIZED
         Owned<IEnvironmentFactory> envFactory = getEnvironmentFactory(true);
         Owned<IConstEnvironment> constEnv = envFactory->openEnvironment();
         Owned<IPropertyTree> root = &constEnv->getPTree();
```

Comment:
these above 3 lines/variables, don't appear to be used.

Reply:
Should be removed.